### ISSUE TITLE
A question about vernalucar names

### ISSUE THREAD

**Issue by punkish** [plazi/treatmentBank#67]
@gsautter 

Consider the following (with `box`, `pageId` and `pageNumber` attributes removed for clarity)

```xml
<subSubSection id="C3336575552A56248876035F1784F3EC" … type="vernacular_names">
<paragraph id="8B9636FE552A56248876035F1784F3EC" …>
<heading id="D0DE8192552A56248876035F1784F3EC" …>Pygmy Hog</vernacularName>
</heading>
</paragraph>
</subSubSection>
…
</paragraph>
</subSubSection>
<subSubSection id="C3336575552A56248BB8031C19B6F3C4" … type="vernacular_names">
<paragraph id="8B9636FE552A56248BB8031C19B6F3C4" …>
<heading id="D0DE8192552A56248BB8031C19B6F3C4" …>
<emphasis id="B95DEAEC552A56248BB8031C1777F3C4" …>French:</emphasis>
<vernacularName id="052A46D0552A56248862031C17EEF3C4" …>Sanglier nain</vernacularName>
/
<emphasis id="B95DEAEC552A562488D7031C167FF3C4" …>German:</emphasis>
<vernacularName id="052A46D0552A5624896B031C16B4F3C4" …>Zwergwildschwein</vernacularName>
/
<emphasis id="B95DEAEC552A5624899A031C1944F3C4" …>Spanish:</emphasis>
<vernacularName id="052A46D0552A5624863E031C1905F3C4" …>Jabali</vernacularName>
pigmeo
</heading>
</paragraph>
</subSubSection>
```

You have a tag called `<subSubSection … type="vernacular_names">` that has one or more children tagged `<vernacularName>`. In one instance, `<vernacularName>` appears only once. In another instance, it appears many times, and is preceded by an `<emphasis>` tag with no semantic attributes, but content that is a language name followed by a ':'. My questions are as follows:

1. Why are the languages not tagged with a semantic tag or attributes?
2. How do you determine the vernacular name/language pair?
3. Is it fair to assume that there is a many-to-many relationship (in a relational db sense) between `treatments` and `vernacularName`?

**Comment by gsautter** [comment]
> * Why are the languages not tagged with a semantic tag or attributes?

Simply because it wasn't a requirement so far, ad we don't have an application for it, either ... much more sensible, and actually preempted in the vernacular names table in the treatment stats is a language field to go with each vernacular name.

> * How do you determine the vernacular name/language pair?

In this case, that was custom built by Connie for the Handbooks of the Mammals of the World ... there is no general-purpose tagger so far, simply because vernacular names are preciously rare in the scientific publications our pipeline is set up to process.

> * Is it fair to assume that there is a many-to-many relationship (in a relational db sense) between `treatments` and `language`:`vernacularName` pair?

It's more between taxon names and vernacular names, but the many-to-many observation is pretty much correct ... most species (at least the ones large enough to be visible to a pair of human eyeballs) do have vernacular names in all the local languages of all the geographic region they appear in ... quite possibly with some species or even genera conflated under one vernacular name in any given place as a result of similar looks.

**Comment by punkish** [comment]
> In this case, that was custom built by Connie for the Handbooks of the Mammals of the World ... there is no general-purpose tagger so far, simply because vernacular names are preciously rare in the scientific publications our pipeline is set up to process.


ah I see. So, the handbook is the only publication with vernacular names so far?

**Comment by myrmoteras** [comment]
There must be more, since we have had Isabelle's project that dealt with this. I also annotated from time to time vernacular names. * How could we find out?
* What are the rules so that they show up in the Stats?

**Comment by gsautter** [comment]
Yes, but the HBMW are the only ones we systematically tagged vernacular names in, far as I remember ... which are the documents Isabelle worked on?

**Comment by myrmoteras** [comment]
Is there a way to find this out checking all the documents?

New species of the day<https://tb.plazi.org/GgServer/static/newToday.html>
________________________________

**Comment by gsautter** [comment]
Not an easy one, I remember getting all the annotation types took several hours ... when we did the latter two months ago, the numbers were 30472 vernacularName annotations in a total of 173 articles ... since I don't think we added too many more since then, this should still be pretty close to the total count ... having indexed only the HBMWs, the SRS stats show 29885 vernacularName annotations in a total of 152 articles (book chapters) with a total of 5787 treatments, with some 600 more hidden in 21 articles that still need to be re-indexed ... quite the needle in a haystack, if a somewhat bigger needle this time.
